store/postgres/groups.go

```diff
@@ -22,11 +24,20 @@ type Group struct {
 	UpdatedAt time.Time `db:"updated_at"`
 }
 
-const (
-	createGroupsQuery = `INSERT INTO groups(name, slug, org_id, metadata) values($1, $2, $3, $4) RETURNING id, name, slug, org_id, metadata, created_at, updated_at;`
-	getGroupsQuery    = `SELECT id, name, slug, org_id, metadata, created_at, updated_at from groups where id=$1;`
-	listGroupsQuery   = `SELECT id, name, slug, org_id, metadata, created_at, updated_at from groups`
-	updateGroupQuery  = `UPDATE groups set name = $2, slug = $3, org_id = $4, metadata = $5, updated_at = now() where id = $1 RETURNING id, name, slug, org_id, metadata, created_at, updated_at;`
+var (
+	createGroupsQuery   = `INSERT INTO groups(name, slug, org_id, metadata) values($1, $2, $3, $4) RETURNING id, name, slug, org_id, metadata, created_at, updated_at;`
+	getGroupsQuery      = `SELECT id, name, slug, org_id, metadata, created_at, updated_at from groups where id=$1;`
+	listGroupsQuery     = `SELECT id, name, slug, org_id, metadata, created_at, updated_at from groups`
+	updateGroupQuery    = `UPDATE groups set name = $2, slug = $3, org_id = $4, metadata = $5, updated_at = now() where id = $1 RETURNING id, name, slug, org_id, metadata, created_at, updated_at;`
+	listGroupUsersQuery = fmt.Sprintf(
+		`SELECT u.id as id, u."name" as name, u.email as email, u.metadata as metadata, u.created_at as created_at, u.updated_at as updated_at
```

Comment:
should we try using query builder here ?

Reply:
sure, we can look into query builder while cleanup and refactoring.